workshops/operations.html

```diff
@@ -238,11 +238,20 @@
 	for details.
       </li>
       <li>
-	Groups that are doing setup and administration for workshops themselves
-	are not expected to pay this fee
+	Self-organised workshps are not expected to pay this fee
 	(but must still abide by
 	<a href="{{page.root}}/faq.html#trademark">these rules</a>
 	if they wish to use the Software Carpentry name and logo).
+        A workshp is considered to be <b>self-organised</b> if the group organising it
+        does all the setup and administration itself, in particular:
+       <ul>
+       <li>finds and communicates with the instructors and helpers without our administrative assistance,</li>
+       <li>sets up and manages participant registration,</li>
+       <li>handles workshop advertisement and provides us with sufficient information to add the workshop
+       to the list of workshops on our website (this is normally done by the lead instructor),</li>
```

Comment:
@katyhuff I agree, thanks. @r-gaia-cs I think @katyhuff is right in this case. Here we provide info for the hosts, the lead instructors have their faqs elsewhere. But anyways thanks for suggestions.